cvat-ui/src/components/annotation-page/review/issues-aggregator.tsx

```diff
@@ -50,7 +53,7 @@ export default function IssueAggregatorComponent(): JSX.Element | null {
         }
 
         return () => {};
-    }, [canvasInstance]);
+    }, [canvasInstance, ready]);
```

Comment:
should we use ``ready`` in the body?

like: ``canvasInstance instanceof Canvas and ready`` ?

Reply:
Added